libs/agno/agno/agent/agent.py

```diff
@@ -9541,7 +9539,7 @@ async def asearch_knowledge_base(query: str, filters: Optional[List[KnowledgeFil
 
         return Function.from_callable(
             search_knowledge_base_function,
-            name="search_knowledge_base_with_agentic_filters",
```

Comment:
why is this changed? isn't this inside the `_search_knowledge_base_with_agentic_filters_function`?